Content.Shared/Buckle/SharedBuckleSystem.cs

```diff
@@ -33,6 +37,18 @@ private void HandleDown(EntityUid uid, SharedBuckleComponent component, DownAtte
             }
         }
 
+        private void HandleMove(EntityUid uid, SharedBuckleComponent component, MovementAttemptEvent args)
+        {
+            if (!component.Buckled) return;
+            args.Cancel();
+        }
+
+        private void HandleChangeDirection(EntityUid uid, SharedBuckleComponent component, ChangeDirectionAttemptEvent args)
+        {
+            if (!component.Buckled) return;
+            args.Cancel();
+        }
```

Comment:
Would be better to probably have a separate "Buckled" component and then use the directed event for it, that way this stuff isn't being called for every single movement / rotation.

Downside is that RemoveComponent is currently bugged via networking so would need to check if it has a client version that runs too.